mxenabled/react-native-widget-sdk
# Fx vulnerabilites

This MR fixes vulnerabilities by running `npm audit fix` and it updates the `CHANGELOG` file.

### Testing instructions

Run the [example app](https://github.com/mxenabled/react-native-widget-sdk/tree/master/example) and ensure everything still works as expected.
